From: Tommy Webb <tommy@calyxinstitute.org>
Date: Wed, 16 Oct 2024 14:27:16 +0000
Subject: Do not use legacy settings order

We don't want to have to keep editing two different XML files,
main_preferences.xml and main_preferences_legacy.xml. So we decide
never to use the legacy ordering.

Change-Id: I5075fd0e7a3a3f57d7b7a4f5ee9940191f864a45
---
 .../src/org/chromium/chrome/browser/settings/MainSettings.java | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/chrome/android/java/src/org/chromium/chrome/browser/settings/MainSettings.java b/chrome/android/java/src/org/chromium/chrome/browser/settings/MainSettings.java
--- a/chrome/android/java/src/org/chromium/chrome/browser/settings/MainSettings.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/settings/MainSettings.java
@@ -701,7 +701,6 @@ public class MainSettings extends ChromeBaseSettingsFragment
     }
 
     private boolean useLegacySettingsOrder() {
-        return !ChromeFeatureList.isEnabled(
-                AutofillFeatures.AUTOFILL_VIRTUAL_VIEW_STRUCTURE_ANDROID);
+        return false;
     }
 }
-- 

